### PR TITLE
Myspace

### PR DESCRIPTION
**Pull request by hswiergiel**

### CODE REVIEW

**Review by SOA-P — DISMISSED** [review]

**Review by Azahires — DISMISSED** [review]

File: backend/src/router.js [hunk 1]
```diff
@@ -14,16 +22,38 @@ const {
...
-  validator.validateUserCreation,
-  hashPassword,
-  userControllers.add
-);
+router.get("/users", userControllers.browse);
+router.get("/users/:id", userControllers.read);
+router.post("/users", validateUserCreation, hashPassword, userControllers.add);
+router.post("/avatar", uploadavatar.single("avatar"), (req, res) => {
+  const { originalname } = req.file;
+  const { filename } = req.file;
```
Comment (LoicBrassart) [inline backend/src/router.js]: const { originalname, filename } = req.file;

File: backend/src/router.js [hunk 2]
```diff
@@ -14,16 +22,38 @@ const {
...
+    `uploads/avatar/${newname}`,
+    (err) => {
+      if (err) throw err;
+      res.send(newname);
+    }
+  );
+});
+router.post("/cv", uploadCV.single("CV"), (req, res) => {
+  const { originalname } = req.file;
+  const { filename } = req.file;
```
Comment (LoicBrassart) [inline backend/src/router.js]: Same

File: frontend/src/components/MSNavbar/style.jsx [hunk 3]
```diff
@@ -1,3 +1,50 @@
...
+  button:hover {
+    color: #c91961;
+  }
+  button:hover:after {
+    width: 100%;
+  }
+  span {
+    position: relative;
+    z-index: 2;
+  } */
```
Comment (LoicBrassart) [inline frontend/src/components/MSNavbar/style.jsx]: Dead code

**Review by LoicBrassart — CHANGES_REQUESTED** [review]
:warning: Beware, add the `uploads` folders to the backend `.gitignore`: I don't need your photos to run this website (and you're lucky you used safe files :smirk: )

File: frontend/src/components/MSNavbar/style.jsx (hunk 3 above)
Comment (hswiergiel) [reply frontend/src/components/MSNavbar/style.jsx]: ok

File: backend/src/router.js (hunk 2 above)
Comment (hswiergiel) [reply backend/src/router.js]: ok

File: backend/src/router.js (hunk 1 above)
Comment (hswiergiel) [reply backend/src/router.js]: ok

**Review by SOA-P — APPROVED** [review]

**Review by LoicBrassart — APPROVED** [review]
Much better!

**Pull request merged**